Description: Fix a non-void function where control can fall off the end.
Origin: https://src.fedoraproject.org/rpms/qepcad-B/blob/91dd25b/f/qepcad-B-return.patch
Forwarded: https://github.com/chriswestbrown/qepcad/pull/2
Last-Update: 2021-06-23

--- a/extensions/sfext/formula/FTYPEINFO.c
+++ b/extensions/sfext/formula/FTYPEINFO.c
@@ -23,6 +23,6 @@
     return TRUE;
   if (FIRST(A) == FALSE)
     return FALSE;
-  
+  FAIL("FTYPEINFO","Unknown formula type!"); 
 }
       
--- a/source/db/SingSacPolicy.h
+++ b/source/db/SingSacPolicy.h
@@ -28,17 +28,17 @@
   Word IPRES(Word r, Word A, Word B)
   {
     if (r > 2)
-      sing->IPRES(r,A,B);
+      return sing->IPRES(r,A,B);
     else
-      sac ->IPRES(r,A,B);
+      return sac ->IPRES(r,A,B);
   }
 
   Word IPDSCR(Word r, Word A)
   {
     if (r > 2)
-      sing->IPDSCR(r,A);
+      return sing->IPDSCR(r,A);
     else
-      sac ->IPDSCR(r,A);
+      return sac ->IPDSCR(r,A);
   }
 
   Word IPFACTGB(Word r, Word I, Word N)  
